### PR TITLE
Clojure file extension fix patch

### PR DESCRIPTION
**Pull request by RohanVashisht1234**

### CODE REVIEW

**Comment by RohanVashisht1234** [comment]
Ok, I have updated clojure file extensions to be correct

**Review by Guldoman — CHANGES_REQUESTED** [review]
This is adding completely irrelevant changes, please revert those.

**Comment by RohanVashisht1234** [comment]
That is due to language_bazel

**Comment by RohanVashisht1234** [comment]
ok, fixed it

**Comment by RohanVashisht1234** [comment]
I think the error has been fixed, but how will I know or test it?

**Review by Guldoman — CHANGES_REQUESTED** [review]
This still contains unrelated changes.

**Comment by RohanVashisht1234** [comment]
Ok, I think the unnecessary changes have been fixed

File: manifest.json [hunk 1]
```diff
@@ -162,7 +162,7 @@
       "id": "meta_languages",
       "mod_version": "3",
       "type": "meta",
-      "version": "0.1.13"
+      "version": "0.1.12"
```
Comment (Guldoman) [inline manifest.json:165]:
```suggestion
      "version": "0.1.13"
```

**Pull request merged**